### DISCUSSION TITLE
Can't get transparent background to work in Pixi.js 7.0+

### DISCUSSION THREAD

**Discussion by jasonoda** [pixijs/pixijs#9425] [Q&A] [answered]
After upgrading from Pixi.js 5.3 to 7.3 the old method of creating a transparent background no longer works. I believe this changed started at 7.0. I'm finding some info about this new PIXI.BackgroundSystem which has an alpha property, but setting it 0 does not change anything. Just seeing a black background. No info out there on google about how to do this in the new version. Can anyone help?

**Comment by SuperSodaSea** [comment, marked as answer]
You need to set an `backgroundAlpha` that is less than 1 when initializing `Application` / `Renderer` to make things work:

```js
const app = new PIXI.Application({
    backgroundAlpha: 0, // Won't work without this line
});
app.renderer.background.alpha = 0.5;
```

Related code:

<https://github.com/pixijs/pixijs/blob/edcda8647da4a4e73877768cac3fbd7cc81920f6/packages/core/src/context/ContextSystem.ts#L205-L219>

**Reply by jasonoda** [reply]
Worked! Thanks SuperSodaSea. Couldn't find the answer anywhere!

**Reply by tisoz** [reply]
> You need to set an that is less than 1 when initializing / to make things work:`backgroundAlpha``Application``Renderer`
> 
> ```js
> const app = new PIXI.Application({
>     backgroundAlpha: 0, // Won't work without this line
> });
> app.renderer.background.alpha = 0.5;
> ```
> 
> Related code:
> 
> https://github.com/pixijs/pixijs/blob/edcda8647da4a4e73877768cac3fbd7cc81920f6/packages/core/src/context/ContextSystem.ts#L205-L219

it's worked , but not running in command:

```
let app = new PIXI.Application({
            width: 400, height: 300

});
app.renderer.background.alpha = 0;
```

I need add `backgroundAlpha: 0` in app init arguments.

`https://pixijs.download/release/docs/PIXI.BackgroundSystem.html`  `alpha` `not working , need old version command` `backgroundAlpha:0`
`https://pixijs.download/release/docs/PIXI.Renderer.html#background`  ` backgroundAlpha` `Deprecated : since 7.0.0`
However, in the official documentation, it is marked as deprecated. At the same time, it doesn't inform us that we need to use backgroundAlpha while using app.renderer.background.alpha. This is very misleading. I hope the official documentation can be updated to address this issue. I've spent an hour trying to find this problem.
